lib/port/manga/manga_kakalot_port.ex

```diff
@@ -0,0 +1,15 @@
+defmodule Ewbscp.Ports.Manga.MangaKakalot do
+  def return_search_results(manga_search_results) do
+    manga_search_results
+    |> Floki.find(".panel_story_list > .story_item > .story_item_right > h3 > a")
+    |> Enum.map(fn {_, [_, {_, manga_uri}], [manga_title]} ->
+     %{manga_uri: manga_uri, manga_title: manga_title} 
```

Comment:
We should return a model established in the domain

Reply:
You're absolutely right. I have some little questions about the architecture. I need to talk with you to solve this questions.

Reply:
I think I get it. Soon I will push some changes and then you'll see if what I did make sense